pyvista/plotting/plotting.py

```diff
@@ -1308,7 +1308,21 @@ def scale(self, scale):
 
     @property
     def camera_position(self):
-        """Return camera position of the active render window."""
+        """Return camera position of the active render window.
+
+        Examples
+        --------
+        Return camera's position and then reposition it.
+
+        >>> mesh = examples.download_bunny_coarse()
```

Comment:
We need to import pyvista here.
```suggestion
        >>> import pyvista as pv
        >>> from pyvista import examples
        >>> mesh = examples.download_bunny_coarse()
```